Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM circleci/elixir:1.6-node
+
+USER root
+
+RUN export DEBIAN_FRONTEND=noninteractive \
+    && apt-get update \
+    && apt-get install -y ffmpeg \
+    && which ffmpeg \
```

Comment:
Do you need this line?

Reply:
Yes. The app calls `ffmpeg` which will fail without this line. This might not matter as much in development, but I want to make sure that this works locally before we start building production images. @jerodsanto was mentioning about eventually removing this dependency, but until that happens, this can act as a reminder.